ores/wsgi/routes/v3/util.py

```diff
@@ -98,8 +98,13 @@ def build_v3_context_model_map(score_request, scoring_system):
 def process_score_request(score_request, scoring_system):
     score_request.model_info = score_request.model_info or ['version']
     try:
-        score_response = scoring_system.score(score_request)
-        return format_v3_score_response(score_response)
+        if len(score_request.rev_ids) > 50:
+            raise ValueError('Should have 50 rev_ids or less in a request.')
```

Comment:
It looks like we want to return an error response.  

See the following line below as an example:
```
return responses.not_found("No scorers available for {0}"
                           .format(e))
```

It looks like `bad_request` is probably what you want to return.  See https://github.com/wikimedia/ores/blob/435e5cc8a4078e35bfbbbd4f2b20a530c9e87db7/ores/wsgi/responses.py#L30 for the definition